app/lib/server/functions/setRealName.js

```diff
@@ -41,7 +41,5 @@ export const _setRealName = function (userId, name, fullUser) {
 };
 
 export const setRealName = RateLimiter.limitFunction(_setRealName, 1, 60000, {
-	0() {
-		return !Meteor.userId() || !hasPermission(Meteor.userId(), 'edit-other-user-info');
-	}, // Administrators have permission to change others names, so don't limit those
+	0() { return !Meteor.userId || !hasPermission(Meteor.userId, 'edit-other-user-info'); }, // Administrators have permission to change others names, so don't limit those
```

Comment:
This doesn't look right. `Meteor.userId` is a function, therefore `!Meteor.userId` is always `false`. The first parameter of `hasPermission(userId, permissionId, scope)` is supposed to be the value returned by calling `Meteor.userId()`.